config/cesm/machines/template.st_archive

```diff
@@ -47,12 +47,20 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
     parser.add_argument("--caseroot", default=os.getcwd(),
                         help="Case directory to build")
 
-    args = CIME.utils.parse_args_and_handle_standard_logging_options(args, parser)
+    parser.add_argument("--no-incomplete-logs", default=False, action="store_true",
+                        help="Whether to archive logs which have been completed or not")
+
+    parser.add_argument("--copy-only", default=False, action="store_true",
```

Comment:
Perhaps someone from CESM can chime in to say if they're interested in these new st_archive features.